packages/ember-toucan-core/src/test-support/components/autocomplete.ts

```diff
@@ -0,0 +1,33 @@
+import { PageObject } from 'fractal-page-object';
+
+export class AutocompletePageObject extends PageObject {
+  get active() {
+    return this.element
```

Comment:
We spread `attributes` on the `<input />`, which seems right to me. However, it means we need to first get ahold of the parent element (`#autocomplete`) before we can select children.